modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md

```diff
@@ -37,8 +37,8 @@ In base SI units this Lagrange multiplier variable has the units $\left[ \frac{A
 where $J$ is the current density, $\sigma$ is the electrical conductivity, and $E$ is the electric field.
 
 
-!alert note title=Employ Consistency in Primary and Secondary Designations
-Consistency in the selection of the primary boundary and secondary boundary among the electrical, thermal, and interface Joule Heating mortar contact input file compoments is recommended.
+!alert note title=Closed Gap Interface Assumed by this Class
+The `ADInterfaceJouleHeatingConstraint` class should only be employed in simulations when the user is certain that the current density like electric potential contact Lagrange multiplier variable is nonzero only when the interface gap is closed. The `ADInterfaceJouleHeatingConstraint` class may also be used in simulations with an open gap at the interface, so long as the electric potential contact Lagrange multiplier variable across that gap is zero while the interface gap is open.
```

Comment:
```suggestion
The `ADInterfaceJouleHeatingConstraint` class should only be employed in simulations when the user is certain that the current-density-like electric potential contact Lagrange multiplier variable is nonzero only when the interface gap is closed. The `ADInterfaceJouleHeatingConstraint` class may also be used in simulations with an open gap at the interface, so long as the electric potential contact Lagrange multiplier variable across that gap is zero while the interface gap is open.
```
Should use hyphens here when the phrase is used as a single adjective modifying "electric potential".

Reply:
I think you've become an NS&T tech editor!

Reply:
It's a blessing and a curse 😂